Registry/sceneassetimporter.setreg

```diff
@@ -8,7 +8,9 @@
 				"SupportedFileTypeExtensions":
 				[
 					".fbx",
-					".stl"
+					".stl",
+					".gltf",
```

Comment:
Do setreg files support comments? Can you maybe add a note that the support here is experimental and use at your own risk? We know there are major issues with GTLF, and our team won't be prioritizing those fixes for some time.

Reply:
I don't know if setreg supports comments. In docs we are trying to avoid using terms like "experimental" so as not to dissuade users from trying the feature and filing issues and PRs. There have also been issues in the past with "experimental" and "preview" labels persisting long past their intended lifespan. The docs page has a "Feature in progress" callout for glTF that explains the feature is in development, and provides links to open PRs and Issues for glTF. I also added a feature request issue that documents my testing with glTF in an attempt to corral glTF discussion and give users some place to go for current info.

Reply:
I'm nervous that enabling this without some heavy markers labeling it as Experimental will result in people trying to use the feature in a production environment. You're right that we often leave things with that label long past their life span, but in this case I think it's appropriate because my team isn't at a point we can support GLTF issues, and until we get our fork of assimp available externally, it will be difficult for other contributors to fix bugs.

Reply:
Comments such as `//` are supported by the Settings Registry. Several other .setreg files also have a bunch of comments. Note though that comments are not standard JSON so some JSON libraries may not be able to process these files in that case.

Reply:
I'm OK with this going in if we agree that any GLTF issues can only be filed at Major or below, and as long as there is no timeline for when they will be addressed, this work likely won't be near the top of my queue for a while.

Reply:
+1 to the above comment. This should be considered "experimental", and maybe should even be hidden behind a feature flag.

Reply:
I'd like to try to change some hearts and minds here before further decisions are made.

Users are asking for this feature. glTF support has issues but there are many scenarios where it works fine, very well even. Users have other options for processing assets for those cases where glTF fails. It's very clear that `.fbx` is the primary transport format for O3DE at this time. Enabling glTF does not make O3DE a worse product; it opens another door for users, Because there are other processing options, I don't think issues around glTF should even rise to major status.

If we enable it without restrictions and without discouraging people from trying it, we increase discoverability and encourage use and improvement from all contributors and users.

Reply:
The bottom line I think is that especially the open-source minded folks have a greater interest in supporting glTF than FBX. The industry as a whole is also slowly moving away from FBX in favor of open standards with wider support and faster growth. Turning on support for glTF may potentially see an increase in support by the community over FBX, simply because several community members have no interest in supporting FBX and/or already have glTF experience.

I don't think deprioritizing glTF problems is the right way to look at the situation. The way I look at it is that there's either a problem with how the data from AssImp is being processed or a specific bug with glTF and AssImp. The former will be more likely, especially since glTF has more extensive test libraries available than FBX. Imho it's not so much about supporting glTF and FBX but rather hardening the AssImp integration by extending the test coverage.

Reply:
Documentation is live: https://o3de.org/docs/user-guide/assets/scene-settings/scene-format-support/